### ISSUE TITLE
[Quest] Mob Mentality

### ISSUE THREAD

**Issue by Ootootoo** [WarEmu/WarBugs#14096]
**Expected behavior and actual behavior:**

The quest is around Badlands for dwarves, and the rewards should be around level 20ish. The rewards of the quest seems to have level 20ish stats, but level 34 requirement

**Steps to reproduce the problem:**

Get the quest "Mob Mentality" in Badlands, order side.

**Screenshots/Videos or archive.org evidences:**

![Valaye_002](https://user-images.githubusercontent.com/50114048/57844384-d9ee9680-77c7-11e9-814d-d49dc146acf5.jpg)

**Comment by wargrimnir** [comment]
These are the correct item level, they're just assigned to the wrong questline. They should have an additional 17 stats that haven't been corrected yet. I stopped mass fixing items once I got to T4 as the value proposition was quite low considering most people go straight for set gear rather than green quest rewards, or were more likely to grind levels to 40.

One day perhaps... but as part of a larger project.
